packages/teams-js/src/public/barCode.ts

```diff
@@ -63,7 +63,12 @@ export namespace barCode {
   /**
    * Checks whether or not media has user permission
    *
-   * @returns true if the user has granted the app permission to media information, false otherwise
+   * @remarks 
+   * For additional information about deivce permissions, see the Learn articles:
+   *  - [Integrate QR or barcode scanner capability](https://learn.microsoft.com/microsoftteams/platform/concepts/device-capabilities/qr-barcode-scanner-capability)
```

Comment:
>Integrate QR or barcode scanner capability [](http://example.com/codeflow?start=9&length=42)

Following this link takes me to a page which provides great information about barcode scanning -- but that page then links to a function in the older (1.x) version of teamsjs and a different capability than this one (it links to `media`) -- is it weird that this capability and function links to docs that don't mention this function at all?